contato.html

```diff
@@ -36,7 +36,7 @@
             </div>
 
             <!-- Menu -->
-            <div class="col-md-6">
+            <div class="col-md-6" id="barra-navegacao-container">
```

Comment:
olha, nao tem problema nenhum vc usar classe aqui e id embaixo, nao precisa mudar, 
mas para fins de organizacao de codigo, eh interessante pensar que quando outra pessoa for ler o seu codigo, vai ficar confuso para ela entender.

Nos nao escrevemos codigo somente para que os programas funcionem, sempre que estiver escrevendo um, pense que escrevemos codigo para outros programadores.

Dito isso, seria legal seguir convencoes. Eh normalmente bem mais facil de entender um site quando o codigo do site inteiro usa a mesma ideia sempre. Por exemplo, sempre defina com id, a nao ser em casos em que queira reusar o estilo, de forma que ele possa ser usado em mais de um lugar. Nesse ultimo caso, use sempre classe entao.